tools/rna_tools/mlocarna/test-data/Makefile

```diff
@@ -0,0 +1,28 @@
+ALL_TESTS=archaea-default archaea-probabilistic haca.snoRNA-default
```

Comment:
`planemo test --conda_dependency_resolution --update_test_data  ./path/to/tool/` generates you test data depending on your test-cases. It is very recommended to check the output carefully afterwards :)